tests/framework/BaseYiiTest.php

```diff
@@ -190,3 +203,11 @@ public function testLog()
         BaseYii::setLogger(null);
     }
 }
+
+class ThemeMock extends Theme
+{
+}
+
+class ViewMock extends View
+{
+}
```

Comment:
I think we should use less complex classes for an example and define configurable fields ourselves. Now this test is dependent not only on DI logic, but on View logic too.

Try something like:

```
class ObjectMock extends BaseObject
{
    public $testField;
}

class SecondMock extends BaseObject
{
}
```